### PR TITLE
Lu 17/drop down/search result

### PR DESCRIPTION
**Pull request by archan96**

### CODE REVIEW

File: src/components/Inputs/SearchInput/components/searchResults/searchResults.styles.ts
Comment (sanketXynes) [inline src/components/Inputs/SearchInput/components/searchResults/searchResults.styles.ts:1]: Is it possible to move this static css to the css file?

**Review by sanketXynes — CHANGES_REQUESTED** [review]
Please update

File: src/components/Inputs/SearchInput/components/searchResults/searchResults.styles.ts
Comment (archan96) [reply src/components/Inputs/SearchInput/components/searchResults/searchResults.styles.ts:1]: Actually as I have created a separate TS file for css so i din't make any more segregation.

**Review by sanketXynes — APPROVED** [review]
Aproved

**Pull request merged**